### PR TITLE
Add blog post for clients

### PR DESCRIPTION
**Pull request by tomicm**

### CODE REVIEW

**Comment by gitbuda** [comment]
Please move the markdown file to a top-level folder because some blog posts might have attached resources. The folder name should match the blog post title. It probably makes sense to rename the file to README.md because GitHub will automatically render the content.

**Review by gitbuda — CHANGES_REQUESTED** [review]
Please move the markdown file to a top-level folder because some blog posts might have attached resources. The folder name should match the blog post title. It probably makes sense to rename the file to `README.md` because GitHub will automatically render the content.

**Comment by gitbuda** [comment]
It also makes sense to use this https://jekyllrb.com/docs/posts as a guide.

**Comment by tomicm** [comment]
I've changed the blog post to Jekyll format after an offline discussion.

File: client-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Official Memgraph Drivers Are Out!
+
+We're happy to announce that Memgraph drivers for Python and C languages, as
+well as Memgraph CLI are now published. You can find their GitHub repositories
```
Comment (karimtrai) [inline client-blog-post.md]: Define (CLI) because it's the first time it's used in the article.

Comment (karimtrai) [inline client-blog-post.md]: Find their respective GitHub repositories here

File: client-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+- [mgclient] (C driver)
+- [pymgclient] (DB-API 2.0 compliant Python driver)
+- [mgconsole] (command line interface)
+
+## How to get started
+
+### Installing mgclient
+
+Memgraph C driver, [mgclient], serves as a base for both [pymgclient] and
```
Comment (karimtrai) [inline client-blog-post.md]: The Memgraph C driver

File: client-blog-post.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+$ pip3 install pymgclient
+```
+
+You can find more detailed installation instructions in the README file in
+[pymgclient] GitHub repository or in the [pymgclient
+documentation](https://memgraph.github.io/pymgclient).
+
+### Installing mgconsole
+
+Memgraph command line interface, [mgconsole], also has to be installed from
```
Comment (karimtrai) [inline client-blog-post.md]: The Memgraph

File: client-blog-post.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+source. The steps are very similar to those for installing the C driver. Again,
+you can find the detailed instructions in the README file in [mgconsole] GitHub
+repository.
+
+
+## Using pymgclient
+
+Python driver is compliant with DB-API 2.0 specification, so using it should
```
Comment (karimtrai) [inline client-blog-post.md]: The Python

File: client-blog-post.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+(:Person {'name': 'Steve'})
+
+# Make database changes persistent
+>>> conn.commit()
+```
+
+
+## Using mgconsole
+
+Here's an example showing how to use Memgraph command line interface:
```
Comment (karimtrai) [inline client-blog-post.md]: The Memgraph command line interface

File: client-blog-post.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+We're happy to announce that Memgraph drivers for Python and C languages, as
+well as Memgraph CLI are now published. You can find their GitHub repositories
+here:
+
+- [mgclient] (C driver)
+- [pymgclient] (DB-API 2.0 compliant Python driver)
+- [mgconsole] (command line interface)
+
+## How to get started
+
```
Comment (karimtrai) [inline client-blog-post.md]: Maybe add a sentence about what's to come next. Something like(e.i. We will now install mgclient, pymgclient and mgconsole and run a few basic queries)

File: client-blog-post.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+| (:Turtle {color: "orange", name: "Mike"}) |
+| (:Turtle {color: "red", name: "Raph"})    |
++-------------------------------------------+
+4 rows in set (0.000 sec)
+memgraph> :quit
+Bye
+```
+
+***
+
```
Comment (karimtrai) [inline client-blog-post.md]: Add a conclusion sentence like (e.i. We have now installed x, y, and z and learned how to ... you are now ready to do x,y and z and here are a few next steps)

File: 2019-05-31-official-memgraph-drivers-are-out.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+well as Memgraph CLI (command line interface) are now published. You can find
+their respective GitHub repositories here:
+
+- [mgclient] (C driver)
+- [pymgclient] (DB-API 2.0 compliant Python driver)
+- [mgconsole] (command line interface)
+
+## How to get started
+
+We will show you how to setup [mgclient], [pymgclient] and [mgconsole] and run
```
Comment (karimtrai) [inline 2019-05-31-official-memgraph-drivers-are-out.md]: We will now learn how to install ...

**Review by gitbuda — APPROVED** [review]

**Pull request merged**